### ISSUE TITLE
Impossible to step into (F11) a function whose name is dynamically evaluated

### ISSUE THREAD

**Issue by jakub-g** [node-inspector/node-inspector#445]
TL;DR it's impossible to step into the function / method whose name is dynamically evaluated (not an identifier / string literal) using F11 or "step into" button. It's possible to do it though when one explicitly puts a breakpoint in that function.

MWE:

Let's consider this simple file `test.js`:

```
var obj = {
  main : function () {
    debugger
    this["my" + "func"]();
  },
  myfunc : function () {
    printHello();
  }
}

function printHello() {
    console.log("Hello")
}

obj.main();
```
# KO scenario
- Run `node --debug-brk test.js`
- Launch `node-inspector` in a separate shell, open it in Chrome
- Press F8 to go to the line with "debugger" statement.
- Press F11, you go to the next line
- Press F11. 

**Expected**: breaking in the first line of "myfunc"
**Actual**: the debugger goes deep through "myfunc" and "printHello" until Console.log!
# OK1 scenario:
- Run `node --debug-brk test.js`
- Put breakpoint on the line containing `console.log("Hello")` and continue the scenario. Now everything is fine.
# OK2 scenario
- Change `this["my" + "func"]` to `this["myfunc"]` and run the scenario. Everything is fine.
# Debugging info

```
$ node-inspector --version
v0.7.4
$ node --version
v0.10.32
```

Running Windows 7 / MINGW.

Also, when tested the code in question in Chrome37@Win7, it worked fine.
